lib/msprime.h

```diff
@@ -123,13 +123,31 @@ typedef struct {
     double c; // constant
 } dirac_coalescent_t;
 
+/* Forward declaration */
+struct _msp_t;
+
 typedef struct {
-    uint32_t locus;
+    double start_frequency;
+    double end_frequency;
+    double alpha;
+    double dt;
+} genic_selection_trajectory_t;
```

Comment:
Different types of trajectories with different parameters are defined here. Question: do we expect there to be different ways of doing this and with different params?

Reply:
yes definitely we should anticipate different kinds. as you aptly named this one, we are doing genic selection, so we can ignore h, the dominance parameter that we might put in to a different kind of trajectory.